subworkflows/local/prokka_cat.nf

```diff
@@ -0,0 +1,34 @@
+//
+// Run PROKKA on contigs that are split by size, then concatenate output and gunzip it
+//
+
+params.prokka_options    = [:]
+params.cat_options       = [:]
+
+include { PROKKA               } from '../../modules/nf-core/modules/prokka/main'           addParams( options: params.prokka_options     )
+include { CAT_CAT as GFF_CAT   } from '../../modules/nf-core/modules/cat/cat/main'          addParams( options: params.cat_options        )
+include { CAT_CAT as FAA_CAT   } from '../../modules/nf-core/modules/cat/cat/main'          addParams( options: params.cat_options        )
+include { CAT_CAT as FNA_CAT   } from '../../modules/nf-core/modules/cat/cat/main'          addParams( options: params.cat_options        )
```

Comment:
If one has different entries for these in `modules.config`, one _can_, but doesn't have to, set params differently.
(I'm not completely up to date on subworkflow syntax though.)

Reply:
Sure, but I'm having a hard time seeing a use case when you want different params for them? I kind of liked having the same since that saves a lot of words..